docs/production/upgrade.md

```diff
@@ -798,26 +754,29 @@ confirm everything is working correctly.
 
 ## Upgrading PostgreSQL
 
-Starting with Zulip 3.0, we use the latest available version of
-PostgreSQL at installation time (currently version 16). Upgrades to
-the version of PostgreSQL are no longer linked to upgrades of the
-distribution; that is, you may opt to upgrade to PostgreSQL 16 while
-running Ubuntu 22.04.
+The major version of PostgreSQL is upgraded separately from the Zulip
+server version. Further, the version of PostgreSQL included with a
+Zulip server is not linked to that of the host OS; the Zulip installer
+uses the latest available version of PostgreSQL at installation time
+(currently, version 16).
 
-Not all versions of Zulip Server support all versions of PostgreSQL, however:
+The following table details which versions each major Zulip Server
+version supports:
 
 ```{include} postgresql-support-table.md
 
 ```
 
 To upgrade the version of PostgreSQL on the Zulip server:
 
-1. Upgrade your server to the latest Zulip release (at least 3.0).
+1. Upgrade your zulip server, at least to the latest Zulip maintenance
```

Comment:
```suggestion
1. Upgrade your Zulip server, at least to the latest Zulip maintenance
```

Reply:
I also don't understand what the "at least" means. Maybe you mean to the last maintenance release in your server version or to the latest Zulip release? Just say it if so.